server/tests/backends/fixtures/replacetext/simple_pypika.json

```diff
@@ -0,0 +1,158 @@
+{
+  "exclude": [
+    "mongo",
+    "pandas",
+    "snowflake"
+  ],
+  "step": {
+    "pipeline": [
+      {
+        "name": "replacetext",
+        "search_column": "beer_kind",
+        "old_str": "Blonde",
+        "new_str": "Deublon"
+      }
```

Comment:
Could you please add a select step to have only relevant columns and make the fixture more readable ?

Reply:
https://github.com/ToucanToco/weaverbird/pull/1529/commits/9ac1e4349bdd1cdb9e09d5e2ff682e3148cc2be8